chunky/src/java/se/llbit/chunky/renderer/scene/PathTracer.java

```diff
@@ -105,7 +105,7 @@ public static boolean pathTrace(Scene scene, Ray ray, WorkerState state, int add
 
       float pSpecular = currentMat.specular;
 
-      double pDiffuse = ray.color.w;
+      double pDiffuse = 1 - Math.sqrt(1 - ray.color.w);
```

Comment:
This has probably come up on Discord before, but is this physically accurate? If not, should we make this change configurable?

Reply:
I think it comes down to interpretation so maybe it wouldn't hurt to add it as an option? But I think it physically makes the most sense, as one would expect 50% opacity to mean 50% of light gets reflected OR absorbed, whereas the way Chunky currently interprets it is "50% gets reflected, 25% gets transmitted, and the remainder is partially transmitted depending on the texture color".